### DISCUSSION TITLE
main branch build error

### DISCUSSION THREAD

**Discussion by C8504** [google/or-tools#4407] [Build and configuration issues]
Hello, do you have this build issue on Mac?
or-tools/ortools/constraint_solver/local_search.cc:3740:19: error: 'filter_stats' in capture list does not name a variable
                [&filter_stats](const LocalSearchFilter* filter1,
Thanks.

**Comment by lperron** [comment]
yes, we know. I will be broken for some time .

**Reply by C8504** [reply]
ok, thanks for reply
